acme/tests/testdata/cert-ipsans.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
`acme/tests/testdata/README` contains commands that were used to generate files. Please consider adding what you used for these files there.

Reply:
Or just describe/make screenshots of the steps you did to create them. I think it's important to have these commands.

Reply:
and it looks like san csr/certs (even domain-only ones) are not documented, should I make those too?

Reply:
Just the ones you created for this PR should be good :)